src/Uno.Templates/content/unoapp/.template.config/template.json

```diff
@@ -422,9 +422,28 @@
     },
     "http": {
       "displayName": "HTTP",
-      "description": "Register native handlers for http and setup Refit support",
+      "description": "Register native handlers for http.",
       "type": "parameter",
-      "datatype": "bool"
+      "datatype": "choice",
+      "defaultValue": "Default",
+      "choices": [
```

Comment:
Since now we have choices (and the user can only choose one option), we probably want to introduce a `None` choice to allow Http to not be used.